Detectors/ITSMFT/MFT/simulation/src/Detector.cxx

```diff
@@ -598,6 +597,116 @@ void Detector::defineSensitiveVolumes()
+  TString sname = mGeometryTGeo->composeSymNameHalf(hf);
+
+  if (!gGeoManager->SetAlignableEntry(sname.Data(), path.Data())) {
+    LOG(FATAL) << "Unable to set alignable entry ! " << sname << " : " << path;
+  }
+
+  Int_t nDisks = mGeometryTGeo->getNumberOfDisksPerHalf(hf);
+
+  for (int dk = 0; dk < nDisks; dk++) {
+    addAlignableVolumesDisk(hf, dk, path, lastUID);
+  }
+}
+
+//_____________________________________________________________________________
+void Detector::addAlignableVolumesDisk(Int_t hf, Int_t dk,
+                                       TString& parent, Int_t& lastUID) const
+{
+  // Add alignable volumes for disk and its daughters
+
+  TString path = Form("%s/%s_%d_%d_%d", parent.Data(), GeometryTGeo::getMFTDiskPattern(), hf, dk, dk);
+  TString sname = mGeometryTGeo->composeSymNameDisk(hf, dk);
+
+  if (!gGeoManager->SetAlignableEntry(sname.Data(), path.Data())) {
+    LOG(FATAL) << "Unable to set alignable entry ! " << sname << " : " << path;
+  }
+
+  Int_t nLadders = 0;
+
+  for (Int_t sensor = mGeometryTGeo->getMinSensorsPerLadder(); sensor < mGeometryTGeo->getMaxSensorsPerLadder() + 1; sensor++) {
+    nLadders += mGeometryTGeo->getNumberOfLaddersPerDisk(hf, dk, sensor);
+  }
+
+  for (Int_t lr = 0; lr < nLadders; lr++) {
+    addAlignableVolumesLadder(hf, dk, lr, path, lastUID);
+  }
+}
+
+//_____________________________________________________________________________
+void Detector::addAlignableVolumesLadder(Int_t hf, Int_t dk, Int_t lr,
+                                         TString& parent, Int_t& lastUID) const
+{
+  // Add alignable volumes for ladder and its daughters
+
+  TString path = parent;
+  path = Form("%s/%s_%d_%d_%d_%d", parent.Data(), GeometryTGeo::getMFTLadderPattern(), hf, dk, lr, lr);
+  TString sname = mGeometryTGeo->composeSymNameLadder(hf, dk, lr);
+
+  LOG(DEBUG) << "Add " << sname << " <-> " << path;
+
+  if (!gGeoManager->SetAlignableEntry(sname.Data(), path.Data())) {
+    LOG(FATAL) << "Unable to set alignable entry ! " << sname << " : " << path;
+  }
+
+  Int_t nSensors = mGeometryTGeo->getNumberOfSensorsPerLadder(hf, dk, lr);
+
+  for (Int_t ms = 0; ms < nSensors; ms++) {
+    addAlignableVolumesChip(hf, dk, lr, ms, path, lastUID);
+  }
+}
+
+//_____________________________________________________________________________
+void Detector::addAlignableVolumesChip(Int_t hf, Int_t dk, Int_t lr, Int_t ms,
+                                       TString& parent, Int_t& lastUID) const
+{
+  // Add alignable volumes for a Chip
+
+  TString path = Form("%s/%s_%d_%d_%d_%d", parent.Data(), GeometryTGeo::getMFTChipPattern(), hf, dk, lr, ms);
+  TString sname = mGeometryTGeo->composeSymNameChip(hf, dk, lr, ms);
+
+  LOG(DEBUG) << "Add " << sname << " <-> " << path;
+
+  if (!gGeoManager->SetAlignableEntry(sname, path.Data(), lastUID++)) {
+    LOG(FATAL) << "Unable to set alignable entry ! " << sname << " : " << path;
```

Comment:
@robincaron13 the CI errors 
````
Error in <TGeoManager::SetAlignableEntry>: A PN entry: has already uid=2
Error in <TGeoManager::SetAlignableEntry>: A PN entry: has already uid=3
Error in <TGeoManager::SetAlignableEntry>: A PN entry: has already uid=4
...
````
are real: you are overwriting the alignable IDs set by the ITS. Please use 
````
int uid = o2::base::GeometryManager::getSensID(o2::detectors::DetID::MFT, chipID);
````
instead of the lastUID. 
For the ITS it works because the it is the detector with ID=0, so the result is the same. I will change it anyway later, to have it explicitly visible.